src/NLog.Web.AspNetCore/README.md

```diff
@@ -0,0 +1,35 @@
+# NLog.Web.AspNetCore
+
+- LoggerProvider for ASP.NET Core platform via extension methods for ILoggingProvider, IWebHostBuilder, and IHostBuilder.
```

Comment:
Think Microsoft calls it [Logging provider](https://learn.microsoft.com/en-us/dotnet/core/extensions/logging-providers) and not LoggerProvider.

Reply:
Completed